ohmec_data_na.js

```diff
@@ -4,6 +4,146 @@ dataNA = {
+        "source":"https://en.wikipedia.org/wiki/Ohlone",
+        "startdatestr":"700:01:01",
+        "enddatestr":"1769:01:01"},
+      "geometry":{
+        "_comment1":"Bay Area",
+        "type":"Polygon",
+        "coordinates":[[[-122.5, 38],[-122.5, 36.5],[-120.5, 36.5],[-120.5, 38]]]}},
+    { "type":"Feature",
+      "id":"ES700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Native American",
+        "entity2type":"people",
+        "entity2name":"Esselen",
+        "labelRotate":45,
+        "labelY":0,
+        "labelX":0,
+        "labelScale":2.0,
+        "fidelity":2,
+        "editdatestr":"2021:09:10",
+        "source":"https://en.wikipedia.org/wiki/Esselen",
+        "startdatestr":"700:01:01",
+        "enddatestr":"1769:01:01"},
+      "geometry":{
+        "_comment1":"Big Sur",
+        "type":"Polygon",
+        "coordinates":[[[-122, 36.5],[-122, 36],[-121.5, 36],[-121.5, 36.5]]]}},
+    { "type":"Feature",
+      "id":"TE700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Native American",
+        "entity2type":"people",
+        "entity2name":"T'epot'aha'l",
+        "labelRotate":45,
+        "labelY":0,
+        "labelX":0,
+        "labelScale":2.0,
+        "fidelity":2,
+        "editdatestr":"2021:09:10",
+        "source":"https://en.wikipedia.org/wiki/Salinan",
+        "startdatestr":"700:01:01",
+        "enddatestr":"1769:01:01"},
+      "geometry":{
+        "_comment1":"Salinas Valley",
+        "type":"Polygon",
+        "coordinates":[[[-121.5, 36.5],[-121.5, 35],[-120.5, 35],[-120.5, 36.5]]]}},
+    { "type":"Feature",
+      "id":"CH700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Native American",
+        "entity2type":"people",
+        "entity2name":"Chumash",
+        "labelRotate":0,
+        "labelY":5,
+        "labelX":0,
+        "labelScale":2.0,
+        "fidelity":2,
+        "editdatestr":"2021:09:10",
+        "source":"https://en.wikipedia.org/wiki/Chumash_people",
+        "startdatestr":"700:01:01",
+        "enddatestr":"1769:01:01"},
+      "geometry":{
+        "_comment1":"Los Angeles",
+        "type":"Polygon",
+        "coordinates":[[[-120.75, 35],[-120.75, 34],[-118, 34],[-118, 35]]]}},
+    { "type":"Feature",
+      "id":"YO700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Native American",
+        "entity2type":"people",
+        "entity2name":"Yokuts",
+        "labelRotate":45,
+        "labelY":0,
+        "labelX":0,
+        "labelScale":2.0,
+        "fidelity":2,
+        "editdatestr":"2021:09:10",
+        "source":"https://en.wikipedia.org/wiki/Yokuts",
+        "startdatestr":"700:01:01",
+        "enddatestr":"1769:01:01"},
+      "geometry":{
+        "_comment1":"Los Angeles",
+        "type":"Polygon",
+        "coordinates":[[[-120.5, 38],[-120.5, 35],[-118, 35],[-118, 38]]]}},
+    // { "type":"Feature",
```

Comment:
Unless you need to refer to these later, I would recommend removing all of the comments.